package com.capg.exceptions;

public class Test4 {
	
	
	 public void division() throws ArithmeticException
	 {
		 System.out.println("division is: " + 4/0);
	 }

}
